signature.c

```diff
@@ -688,12 +690,12 @@ static int pv_signature_validate_pk(struct mbedtls_pk_context *pk)
 	return ret;
 }
 
-static bool pv_signature_verify_sha(const char *payload,
-				    struct dl_list *certs_raw,
-				    struct pv_signature *signature,
-				    mbedtls_md_type_t mdtype)
+static sign_file_res_t pv_signature_verify_sha(const char *payload,
+					       struct dl_list *certs_raw,
+					       struct pv_signature *signature,
+					       mbedtls_md_type_t mdtype)
 {
-	bool ret = false;
+	bool ret = SIGN_FILE_NOK_SHA;
```

Comment:
This is weird, this bool should be `sign_file_res_t` ?

Reply:
that's bad. Good catch :)